spec/features/users_show_spec.rb

```diff
@@ -26,6 +26,10 @@
     expect(page).to have_content('Post 2')
     expect(page).to have_content('Post 3')
   end
+
+  scenario 'asserts you can see the number of posts the user has written' do
+    expect(page).to have_content('3 Posts')
+  end
```

Comment:
Nice job adding the test cases requested by the previous reviewer😉. Kindly note that the test cases you have added are failing and it is a requirement that all tests must be passing👍🏼.

to fix this one, see the suggestion below.

```
  scenario 'asserts you can see the number of posts the user has written' do
    expect(page).to have_content('Number of posts: 3')
  end
```

Kindly do well to fix the pagination test also😉.

<img width="736" alt="image" src="https://github.com/DevaWinner/blog-app/assets/64643995/e6998413-7216-4b91-838e-7ca630d83c31">

Reply:
Thanks for the suggestion. I have implemented it and there is no failure